manifests/base/tasks/kustomization.yaml

```diff
@@ -11,3 +11,9 @@ commonLabels:
 images:
 - name: example
   newName: quay.io/operate-first/opf-toolbox:v0.5.0
+configMapGenerator:
+- name: robozome-configMap
```

Comment:
We already know the resource is a configmap, no need to have a configmap resource named `robozome-configMap`, let's remove the redundant `-configMap` from the name.